worlds/tunic/__init__.py

```diff
@@ -297,8 +324,50 @@ def remove_filler(amount: int) -> None:
             if tunic_item.name in slot_data_item_names:
                 self.slot_data_items.append(tunic_item)
 
+        # pull out the filler so that we can place it manually during pre_fill
+        self.local_filler = []
+        if self.options.grass_randomizer and self.options.grass_fill > 0 and self.multiworld.players > 1:
+            # skip items marked local or non-local, let fill deal with them in its own way
```

Comment:
Have you considered adding something like:
```suggestion
            # skip items marked local or non-local, let fill deal with them in its own way
            # remove grass from non_local if it's meant to be limited
            if self.settings.limit_grass_rando:
                self.options.non_local_items.value.remove("Grass")
```

Reply:
Oh wow, did I really suggest `remove` 🤦 That should be `discard("Grass")`

Reply:
Good catch, I wasn't aware of discard